### PR TITLE
Add renderHeader which acts the same as renderFooter but at the top

### LINKED ISSUE #86: Implement renderHeader method
It would be really nice if library would implement `renderHeader` witch would work exactly like `renderFooter` method.

### PR DESCRIPTION
**Pull request by jcurtis**
Helps a bit with #89 
Fixes #86 

Reposted this due to branching change

### CODE REVIEW

File: src/SortableList.js [hunk 1]
```diff
@@ -222,10 +229,10 @@ export default class SortableList extends Component {
       if (rowsLayouts) {
         if (horizontal) {
           location.x = nextX;
-          nextX += rowsLayouts[key].width;
+          nextX += rowsLayouts[key] ? rowsLayouts[key].width : 0;
```
Comment (gitim) [inline src/SortableList.js]: do we really need this check?

File: src/SortableList.js [hunk 2]
```diff
@@ -178,7 +184,7 @@ export default class SortableList extends Component {
     const containerStyle = StyleSheet.flatten([style, {opacity: Number(animated)}])
     const innerContainerStyle = [
       styles.rowsContainer,
-      horizontal ? {width: contentWidth} : {height: contentHeight},
+      horizontal ? {width: contentWidth} : {height: contentHeight}
```
Comment (gitim) [inline src/SortableList.js]: lets keep trailing comma

File: src/SortableList.js (hunk 1 above)
Comment (jcurtis) [reply src/SortableList.js]: Yes - I ran into crashes using react-navigation when this component was not currently on the screen but being updated. I probably could have made a separate PR for that. Let me know if you'd prefer that.

Comment (gitim) [reply src/SortableList.js]: yea, separate pr will be better

Comment (jcurtis) [reply src/SortableList.js]: Coming right up.

Comment (jcurtis) [reply src/SortableList.js]: Moved fix over to #93

**Comment by gitim** [comment]
ok, thanks! will release it soon

**Pull request merged**